src/pybind/mgr/dashboard/frontend/cypress.json

```
@@ -8,5 +8,9 @@
   "defaultCommandTimeout": 120000,
   "viewportHeight": 1080,
   "viewportWidth": 1920,
-  "projectId": "k7ab29"
+  "projectId": "k7ab29",
+  "reporter": "cypress-multi-reporters",
+  "reporterOptions": {
+    "configFile": "reporter-config.json"
```

Why not set the configuration directly here?
```suggestion
      "reporterEnabled": "spec, mocha-junit-reporter",
      "mochaJunitReporterReporterOptions": {
        "mochaFile": "cypress/results/results-[hash].xml"
  }
```

---

Tried but the `mocha-multi-reporters` & `cypress-multi-reporters` syntax was a bit confusing... Now it works.